feat: add organization authentication notes for Connected Accounts flow (#1041)
auth0/docs-v2: auth4genai/snippets/integrations/next-step.mdx

## Next steps

- To learn how to configure applications to access Token Vault, read [Configure Token Vault](https://auth0.com/docs/secure/tokens/token-vault/configure-token-vault).

> - If your application uses [Organizations](https://auth0.com/docs/manage-users/organizations), sign the user in to the correct organization before you initiate the Connected Accounts flow. Connected accounts remain tied to the individual user profile.

Reviewer: ```suggestion
- If your application uses [Organizations](https://auth0.com/docs/manage-users/organizations), authenticate the user with the target organization before initiating the Connected Accounts flow. Connected accounts remain linked to the individual user profile.
```
